.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+cache: pip
+dist: xenial
+
+matrix:
+  include:
+    - name: "shell syntax"
+      script: find . -path "./submodules" -prune -o -type f -name "*.sh" -print | xargs -n1 shellcheck -x --format=gcc
+    - name: "playbook syntax"
+      script: ansible-lint */*.yml
```

Comment:
I think this should be done on a case by case basis instead of massively, but I suppose that's why you added allow_failure on playbook syntax.

Sounds a good first start.

Reply:
Indeed, we should strive to fix this (and shell) in small batches, thats why its disabled, so we can enable at a later date when everything is fixed so we dont introduce new broken stuff